indexer/services/ender/src/validators/yields-params-validator.ts

```
@@ -1,25 +1,25 @@
-        'UpdateYieldParamsEvent must have an assetYieldIndex that is defined and non-empty',
+        'UpdateYieldsParamsEvent must have an assetYieldsIndex that is defined and non-empty',
         { event: this.event },
       );
     }
 
     if (this.event.sdaiPrice === undefined || this.event.sdaiPrice === '') {
       return this.logAndThrowParseMessageError(
-        'UpdateYieldParamsEvent must have an sDAIPrice that is defined and non-empty',
+        'UpdateYieldsParamsEvent must have an sDAIPrice that is defined and non-empty',
         { event: this.event },
       );
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)